config/jobs/kubernetes-sigs/kind/kind.yaml

```diff
@@ -7,7 +7,6 @@ periodics:
     repo: kind
     base_ref: main
     path_alias: sigs.k8s.io/kind
-  always_run: true
```

Comment:
this is a valid field

Reply:
no

Reply:
maybe the config check is outdated?
This and `optional` are valid.

Reply:
I think that is valid because there are thousands of those fields:

```
 grep -r always_run config/jobs/ | wc
   1136    3543  115004
```

Also, I don't think we should rename jobs in this PR

Reply:
ahh, ok, I understand it now `always_run` only applies to presubmit jobs, it is correct

The comment about renaming stands, but let me ask @BenTheElder